### DISCUSSION TITLE
How will HTML export work?

### DISCUSSION THREAD

**Discussion by tm-a-t** [typst/typst#2597] [General]
There is HTML export feature in the roadmap and [a related RFC issue.](https://github.com/typst/typst/issues/721) Does the feature imply rendering a Typst document as a regular HTML page with KaTeX-like formulas? Is it right that the feature is in the stage of early discussion and shouldn’t be expected in 2024?

**Comment by laurmaedje** [comment]
It will be a regular web page with MathML formulas. It's still early and shouldn't be expected in 2023, but we do plan to make it happen in 2024.
